pkg/network/ebpf/c/protocols/http.h

```diff
@@ -2,7 +2,7 @@
 #define __HTTP_H
 
 #include "bpf_builtins.h"
-#include "tracer.h"
+#include "../tracer.h"
```

Comment:
I don't *think* you need to change this, because we specify `pkg/network/ebpf/c` as an include directory during the build

Reply:
![image](https://user-images.githubusercontent.com/17148247/198837571-12f0f757-1ff6-4c29-8d69-44b4722971ef.png)

Reply:
that's after me removing it

Reply:
fixed by changing the `system-probe` task